### PR TITLE
WIP Google Cloud config

### PR DESCRIPTION
**Pull request by englishm**
container build + manifests for a semi manual k8s deployment in Google Cloud

(Builds on work started in #49 )

### CODE REVIEW

File: manifests/moq-demo-deploy.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    metadata:
+      creationTimestamp: null
+      labels:
+        app: moq-demo
+    spec:
+      containers:
+      - env:
+        - name: RUST_LOG
+          value: info
+        image: us-west1-docker.pkg.dev/quic-video/moq/moq-demo:v4
```
Comment (kixelated) [inline manifests/moq-demo-deploy.yaml:27]: How do we deploy new versions?

**Review by kixelated — COMMENTED** [review]
See comments on #49

**Comment by englishm** [comment]
(just recently rebased, still need to re-test)

File: manifests/moq-demo-deploy.yaml (hunk 1 above)
Comment (englishm) [reply manifests/moq-demo-deploy.yaml:27]: lol, right now just by manually editing this Deployment with `kubectl`. It's a bit jank still, for sure.

**Comment by englishm** [comment]
Had this running in GKE at one point, but @kixelated is working on a different Docker+VM-based setup now, so closing this.

**Pull request closed without merging**